pkg/register/component.go

```diff
@@ -35,30 +36,25 @@ type ConfigurationDBModule interface {
 
 // Component is the register component interface.
 type Component interface {
-	RegisterUser(ctx context.Context, realmName string, user apiregister.User) (string, error)
-	GetConfiguration(ctx context.Context, realmName string) (apiregister.Configuration, error)
+	RegisterUser(ctx context.Context, socialRealmName, clientRealmName string, user apiregister.UserRepresentation) (string, error)
```

Comment:
socialRealmName is a fixed value so I think we could put it in the component struct

Reply:
Rolled back